web/src/message_list_view.ts

```diff
@@ -651,6 +654,8 @@ export class MessageListView {
         sender_is_guest: boolean;
         should_add_guest_indicator_for_sender: boolean;
         is_hidden: boolean;
+        // TODO: Ideally this wouldn't be both nullable and possibly undefined.
+        // Look into what would be involved in changing that.
         mention_classname?: string | null;
```

Comment:
There's not a lot of references, probably best to just do this rather than adding a TODO.

```
$ git grep mention_classname
web/src/compose_fade.ts:    mention_classname: string | null;
web/src/message_list_view.js:        let mention_classname;
web/src/message_list_view.js:                    mention_classname = "direct_mention";
web/src/message_list_view.js:                mention_classname = "group_mention";
web/src/message_list_view.js:            mention_classname = null;
web/src/message_list_view.js:            ...(mention_classname !== undefined && {mention_classname}),
web/templates/single_message.hbs:  class="message_row{{#unless msg/is_stream}} private-message{{/unless}}{{#include_sender}} messagebox-includes-sender{{/include_sender}}{{#if mention_classname}} {{mention_classname}}{{/if}}{{#msg.unread}} unread{{/msg.unread}} {{#if msg.locally_echoed}}locally-echoed{{/if}} selectable_row"
web/tests/message_list_view.test.js:        assert.equal(result[1].mention_classname, "group_mention");
web/tests/message_list_view.test.js:        assert.equal(result[1].mention_classname, null);
web/tests/message_list_view.test.js:        assert.equal(result[1].mention_classname, "group_mention");
web/tests/message_list_view.test.js:        assert.equal(result[1].mention_classname, null);
```

Reply:
done!